.rubocop_todo.yml

```diff
@@ -0,0 +1,235 @@
+# This configuration was generated by
```

Comment:
This file should probably not be here... as in not submitted to version control.